sig-docs/installation/community-collaboration-for-v3.0.0.md

```diff
@@ -28,7 +28,7 @@ If you would like to contribute to community, raise a PR to add yourself as an a
 | --- | --- | --- | --- | --- | --- |
 | # 14 | KubeSphere on AWS EKS | TBD  | TBD | TBD |
 | # 15 | KubeSphere on Aliyun ACK | [Pixiake](https://github.com/pixiake) | Aug, 13 | English & Chinese | TBD |
-| # 16 | KubeSphere on Azure AKS | TBD  | TBD | TBD | TBD |
+| # 16 | KubeSphere on Azure AKS | [Roland Ma](https://github.com/RolandMa1986)   | TBD | TBD | TBD |
```

Comment:
Please help to add the `Target Date` and `Preferred Language` in this table. 

BTW, could you please also take the `task #3 | KubeSphere on Azure VM`?

Reply:
Ok, I can work on it. Is there a OS requirement while using VM?

Reply:
Yes, we will provide a document template with you later.